src/AppBundle/Form/Type/TaskType.php

```diff
@@ -52,7 +52,14 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('tags', 'tag', [
                 'required' => false,
-                'choices'  => $this->taskwarrior->tags()
+                'choices'  => $this->taskmanager->getTaskwarrior()->tags()
+            ])
+            ->add('dependencies', DependencyType::class, [
+                'required' => false,
+                'choices' => $this->taskmanager->createQueryBuilder()
```

Comment:
Most definitely. I just adhere to MVP development here. First step is to expose dependencies UI. The next would be to switch to AJAX search & add via choices choice_loader but after dependency management works at all. 

That's why I marked the commit as WIP ;-) I wanted to hear your thoughts and move things one at a time.